### PR TITLE
Use (short) version string in package version history

### LINKED ISSUE #5651: semver2 version's metadata should be displayed in a separate column
displaying the metadata with the version in the same column as version gets ugly when the metadata gets long. My suggestion is to have a separate column for version metadata if it is a semver2 version.

### PR DESCRIPTION
**Pull request by zivkan**
fixes #5651

### CODE REVIEW

**Comment by agr** [comment]
How about some screenshots?

File: src/NuGetGallery/Views/Packages/DisplayPackage.cshtml [hunk 1]
```diff
@@ -469,9 +469,9 @@
                                         }
                                     </td>
                                     <td>
-                                        <a href="@Url.Package(packageVersion)" title="@packageVersion.FullVersion">
+                                        <a href="@Url.Package(packageVersion)" title="@packageVersion.Version">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: `title` is a mouse-over so we can be more verbose right?

Comment (zivkan) [reply src/NuGetGallery/Views/Packages/DisplayPackage.cshtml]: yes, you're right. I'll change it back.

**Comment by zivkan** [comment]
Screenshot after changes. See #5651 for a similar screenshot without these changes.

image

**Pull request merged**